Update of CCM Compatibility Matrix and Outdated Info on Page (#332)
chocolatey/docs: input/en-us/central-management/index.md

The following are all of the Chocolatey components required for Central Management to work.

> * Chocolatey (`chocolatey` package)

Reviewer: ```suggestion
* Chocolatey CLI (`chocolatey` package)
```

* Chocolatey for Business (C4B) Edition.
* Chocolatey Licensed Extension (`chocolatey.extension` package)
* Chocolatey Agent (`chocolatey-agent` package)

> * CCM Database (`chocolatey-management-database` package)

Reviewer: ```suggestion
* Chocolatey Central Management Database (`chocolatey-management-database` package)
```

  * This deploys the CCM database schema to a specified SQL Server instance

> * CCM Service (`chocolatey-management-service` package)

Reviewer: ```suggestion
* Chocolatey Central Management Service (`chocolatey-management-service` package)
```

  * This installs the CCM Service, which the Chocolatey Agent will communicate with.

> * CCM Website (`chocolatey-management-web` package)

Reviewer: ```suggestion
* Chocolatey Central Management Website (`chocolatey-management-web` package)
```

>   * This is the CCM front end website that is the main user interface of the application

Reviewer: ```suggestion
  * This is the Chocolatey Central Management front-end website that is the main user interface of the application.
```

> ### CCM Component Compatibility Matrix

Reviewer: ```suggestion
### Chocolatey Central Management Component Compatibility Matrix
```

The following are all of the Chocolatey components required for Central Management to work.

* Chocolatey CLI (`chocolatey` package)
* Chocolatey for Business (C4B) Edition.
* Chocolatey Licensed Extension (`chocolatey.extension` package)
* Chocolatey Agent (`chocolatey-agent` package)
* Chocolatey Central Management Database (`chocolatey-management-database` package)
  * This deploys the Chocolatey Central Management database schema to a specified SQL Server instance.
* Chocolatey Central Management Service (`chocolatey-management-service` package)
  * This installs the Chocolatey Central Management Service, which the Chocolatey Agent will communicate with.
* Chocolatey Central Management Website (`chocolatey-management-web` package)
  * This is the Chocolatey Central Management front-end website that is the main user interface of the application.

### Chocolatey Central Managment Component Compatibility Matrix

> Central Management has specific needs that are mostly handled by packaging aspects. As the Chocolatey Agent and Central Management communicate with each other, there are some versions that may not be compatible with each other due to mistakes or fixes that needed to be implemented. This serves as a means of capturing that for you.

Reviewer: ```suggestion
Chocolatey Central Management has specific needs that are mostly handled by packaging aspects. As the Chocolatey Agent and Chocolatey Central Management communicate with each other, there are some versions that may not be compatible with each other due to mistakes or fixes that needed to be implemented. This serves as a means of capturing that for you.
```

> :memo: **NOTE**
>

> > Central Management packages (all three) are treated as a singular unit, meaning all three packages across one or more machines must all be on the same version. Using different versions of Central Management packages (db, service, web) is completely unsupported and likely will not even work properly.

Reviewer: ```suggestion
> Chocolatey Central Management packages are treated as a singular unit, meaning that they all must all be on the same version across one or more machines. Using different versions of Chocolatey Central Management packages is unsupported and will likely not work properly.
```

Reviewer: I removed the reference to the individual packages or 'three' so that we don't have to revisit this in future if we ever change the number / type of them.

Author: makes sense. good suggestion

## Stay Up To Date

> * [Central Management Release Notes](xref:ccm-release-notes)

Reviewer: ```suggestion
* [Chocolatey Central Management Release Notes](xref:ccm-release-notes)
```

![CCM Overview](/assets/images/features/ccm/ccm_overview.jpg)

> ### Central Management API

Reviewer: ```suggestion
### Chocolatey Central Management API
```

> * [CCM API](xref:ccm-api)

Reviewer: ```suggestion
* [Chocolatey Central Management API](xref:ccm-api)
```

### ChocoCCM Powershell Module

* [ChocoCCM Module](xref:chococcm)

## Related Articles

> * [C4B Quick Start Guide(QSG)](xref:c4b-quick-start-guide)

Reviewer: ```suggestion
* [Chocolatey For Business Quick Start Guide(QSG)](xref:c4b-quick-start-guide)
```

> * [C4B Azure Enviornment](xref:qdeazure)

Reviewer: ```suggestion
* [Chocolatey For Business Azure Enviornment](xref:qdeazure)
```

## Roadmap

> * [CCM Development Roadmap](xref:roadmap#chocolatey-central-management)

Reviewer: ```suggestion
* [Chocolatey Central Management Development Roadmap](xref:roadmap#chocolatey-central-management)
```

### I'm a licensed customer, now what?

See [Getting Chococlatey Central Management](#getting-ccm).

### Will this become available for lower editions of Chocolatey?

> CCM will only be available in Chocolatey for Business (C4B).

Reviewer: ```suggestion
Chocolatey Central Management will only be available in Chocolatey for Business (C4B).
```

Reviewer: I know this was one of your changes but I saw it and we should update it for consistency.

> ### What's the minimum version of the Chocolatey packages I need to use CCM?

Reviewer: ```suggestion
### What's the minimum version of the Chocolatey packages I need to use Chocolatey Central Management?
```

### What's the minimum version of the Chocolatey packages I need to use CCM?

> See [CCM Component Compatibility Matrix](#ccm-component-compatibility-matrix).

Reviewer: ```suggestion
See [Chocolatey Central Management Component Compatibility Matrix](#ccm-component-compatibility-matrix).
```

### Where can I find the changelog or release notes for Chocolatey Central Management?

Please see [Central Management Release Notes](xref:ccm-release-notes).

### How do I get support?

Please run `choco support` from a licensed edition and follow the instructions.

### How do I set up Chocolatey Central Management?

> * You can standup CCM from one powershell script as part of our [C4B Quick Start Guide](xref:c4b-quick-start-guide). Allowing you setup an entire Chococlatey enviornment in just 6 simple steps.

Reviewer 2: ```suggestion
* As part of our [C4B Quick Start Guide](xref:c4b-quick-start-guide)
```

> * CCM is stood up as part of our [C4B Azure Enviornment](xref:qdeazure)

Reviewer 2: ```suggestion
* As part of our [C4B Azure Enviornment](xref:qdeazure)
```

> * You can also setup CCM on its own following the [CCM Setup Docs](xref:ccm-setup)

Reviewer 2: ```suggestion
* By following the [CCM Setup Docs](xref:ccm-setup)
```

### Where can I find the changelog or release notes for Chocolatey Central Management?

> Please see [Central Management Release Notes](xref:ccm-release-notes).

Reviewer: ```suggestion
Please see [Chocolatey Central Management Release Notes](xref:ccm-release-notes).
```

### How do I get support?

Please run `choco support` from a licensed edition and follow the instructions.

### How do I set up Chocolatey Central Management?

> * As part of our [C4B Quick Start Guide](xref:c4b-quick-start-guide). Allowing you setup an entire Chococlatey enviornment in just 6 simple steps.
> * As part of our [C4B Azure Enviornment](xref:qdeazure)
> * By following the [CCM Setup Docs](xref:ccm-setup)

Reviewer: ```suggestion
* As part of our [Chocolatey For Business Quick Start Guide](xref:c4b-quick-start-guide). Allowing you setup an entire Chococlatey enviornment in just 6 simple steps.
* As part of our [Chocolatey For Business Azure Enviornment](xref:qdeazure)
* By following the [Chocolatey Central Management Setup Docs](xref:ccm-setup)
```
